cdk/app/chart.py

```diff
@@ -132,9 +133,9 @@ def _create_deployment(self):
 
         self._add_labels(deployment.metadata)
         self._add_labels(deployment.pod_metadata)
-        deployment.pod_metadata.add_label("component", "local-office-search-api-server")
-        deployment.metadata.add_annotation(
-            "ad.datadoghq.com/local-office-search-api-server.logs",
+        deployment.pod_metadata.add_label("component", container_name)
+        deployment.pod_metadata.add_annotation(
```

Comment:
Is this the only line that has really changed? (Along with line 187, which is the equivalent line for the scheduled job.) How does the fix work?